### PR TITLE
New sand ruin - the E-11 Manufacturing Plant.

### PR DESCRIPTION
**Pull request by Sadhorizon**
## About The Pull Request
An old Eoehoma manufacturing plant dedicated to assembly of the beloved-by-many E-11 rifle. It has since become home to a particularly dedicated group of hermits.

**TODO:**
- [x] stuff outside the fence (placeholder rn)
- [x] fill out all the descs
- [x] beg people for E-11 inhands (maybe) (just put placeholders there, might be good enough)
- [x] rust because I forgor
- [x] faction for the hermits

<details><summary>Screenshots:</summary>
<p>

scrnshot1

StrongDMM-2024-09-27 20 58 55

StrongDMM-2024-08-31 13 53 44

</p>
</details> 

Loot you can expect here is: Way Too Many E-11s (~12); Cargo mech; Cash; Some mats; One E-10; One E-60; Six Rations

Danger you can expect here: One Hermit Soldier; One Hermit Hunter; Two Hermit Wanderers; Eleven Hermit Troopers; One Goliath; One Legion; A bunch of bear traps; Acid pit I guess?

Also adds: E-11 /no_cell variant; E-11 hermits; Purple hardhat (need it ok?).

## Why It's Good For The Game
Sand planet is like. Medipen ruin 50% of the time, I swear.
Also uh, Eoehoma love I guess.

## Changelog

:cl:
add: A new sand ruin - the E-11 Manufacturing Plant - and associated code.
add: A purple hardhat.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/human/survivors.dm [hunk 1]
```diff
@@ -67,6 +67,21 @@
...
 	)
 
+/mob/living/simple_animal/hostile/human/hermit/ranged/e11 // Intended for the e11_manufactory ruin.
+	name = "Hermit Trooper"
+	desc = "Quality weapons are hard to get by in the sandworlds, which forces many survivors to improvise with that they have. This one is hoping that an E-11 of all things will save his life."
+	icon_state = "survivor_e11"
+	icon_living = "survivor_e11"
+	projectilesound = 'sound/weapons/gun/laser/e-fire.ogg'
+	speed = 10
+	faction = list("eoehoma")
```
Comment (FalloutFalcon) [inline code/modules/mob/living/simple_animal/hostile/human/survivors.dm:78]: someone should really make this a define.

**Review by FalloutFalcon — APPROVED** [review]

File: code/modules/mob/living/simple_animal/hostile/human/survivors.dm (hunk 1 above)
Comment (Sadhorizon) [reply code/modules/mob/living/simple_animal/hostile/human/survivors.dm:78]: Sorry, make what a define? Faction?

**Review by meemofcourse — CHANGES_REQUESTED** [review]
imagen
i know outside the buildings aren't that important, but these places feel really empty. if you want you can either shrink the map or add stuff or further decaling to these. maybe there's some small temporary cargo sites with crates around that you can open for maint loot? wrecked ripleys? blood and Mysterious Goo? watch towers in the corners of the site? this really makes me wish we had cargo containers

imagen
this reception area could use some folding cabinets, a bin, and a water cooler

imagen
this section is empty. maybe add some, like. beer, and acid in the glass would be funny

imagen
if you're going to place traps for the fauna, you should probably make them higher in quantity

imagen
i like this. you should add some mushrooms

imagen
:(

imagen
this airlock doesn't actually work, and immediately depressurizes the area when opened. also anchor the gun locker (and a lot of the other lockers too)

cool ruin

**Review by meemofcourse — APPROVED** [review]
lets go tm this i suppose

**Comment by Gristlebee** [comment]
Sighs so bigly.

**Pull request merged**